.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We do not require learners to use vscode for this tutorial, so I don't think it makes sense to include this vscode task in the repo.